src/activities/UseFmeService.ts

```diff
@@ -41,9 +232,16 @@ export interface UseFmeServiceInputs {
         | "application/json"
         | "multipart/form-data";
     /**
-     * @description The body parameters to pass to the service operation when the method is POST or PUT.
+     * @description The body parameters or object to pass to the service operation when the method is POST or PUT.
      */
     parameters?: string | Record<string, any>;
+    /**
+     * @description Parameters used to substitute {token} values in the path.
```

Comment:
Maybe this would benefit from an example to help illustrate the syntax.

Reply:
Done

Reply:
Nice, thanks!